pkg/controller/mobiledeveloperconsole/mobiledeveloperconsole_controller.go

```diff
@@ -516,7 +517,30 @@ func (r *ReconcileMobileDeveloperConsole) Reconcile(request reconcile.Request) (
 	}
 
 	//## endregion GrafanaDasboard
-	//#endregion
+
+	//#region mobile-developer Role
+	mobileDeveloperRole := &rbacv1.Role{ObjectMeta: metav1.ObjectMeta{Name: "mobile-developer", Namespace: instance.Namespace}}
+	op, err := controllerutil.CreateOrUpdate(context.TODO(), r.client, mobileDeveloperRole, func(ignore runtime.Object) error {
+		reconcileMobileDeveloperRole(mobileDeveloperRole)
+		return nil
+	})
+	if err != nil {
+		return reconcile.Result{}, err
+	}
+	reqLogger.Info("Role reconciled:", "Role.Name", mobileDeveloperRole.Name, "Role.Namespace", mobileDeveloperRole.Namespace, "Operation", op)
```

Comment:
We could wrap this in a check, and only log if the operation result is "created" or "updated" and not "unchanged". That should reduce the amount of unnecessary log messages. Thoughs?